pontoon/base/apps.py

```diff
@@ -16,6 +16,9 @@ def ready(self):
         super(BaseConfig, self).ready()
         self.monkeypatch()
 
+        # Load celery app so shared_task uses it for executing tasks.
```

Comment:
Yeah, celery.shared_task would help a lot.

I was afraid shared_task might confuse a reader, especially if she's not in the process of reviewing this particuar PR.